djangocms_navigation/migrations/0008_language_migration.py

```diff
@@ -0,0 +1,35 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+import logging
+
+from django.db import migrations
+
+from cms.utils.i18n import get_default_language_for_site
+
+logger = logging.getLogger(__name__)
+
+
+def add_navigation_languages(apps, schema_editor):
+    db_alias = schema_editor.connection.alias
+    navigation_queryset = apps.get_model("djangocms_navigation", "MenuContent").using(db_alias).filter(language_exact="")

```

Comment:
This is erroring: 

```
  File "/virtualenv/lib/python3.6/site-packages/djangocms_navigation/migrations/0008_language_migration.py", line 15, in add_navigation_languages
    navigation_queryset = apps.get_model("djangocms_navigation", "MenuContent").using(db_alias).filter(language_exact="")
AttributeError: type object 'MenuContent' has no attribute 'using'
```

Reply:
Try and be a little cleaner and explicit with this, what I mean here is to make it easier to digest what is happening rather than trying to do it all on one line:

```
MenuContent = apps.get_model("djangocms_navigation", "MenuContent")
navigation_queryset = MenuContent.objects.using(db_alias).filter(language_exact="")
```

Notice the `.objects.using(` it was the missing objects that threw the error.